docs/modules/deployment/pages/time-series-storage/timeseries/hosted-tss.adoc

```diff
@@ -43,13 +43,13 @@ echo "opennms-plugin-cloud-core wait-for-kar=opennms-plugin-cloud" | sudo tee ${
 
 === Configure Cloud Services
 
-You must configure your {page-component-title} instance to load the Cloud Services plugin on start, and to direct storage to the cloud service.
-If you are using Horizon 31 or later, copy the example `timeseries.properties` file to your working directory:
+You must configure your {page-component-title} instance to direct storage to the cloud service.
+If you are using Horizon 31 or later add or update the following line in `$\{OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties`:
 
 [source, console]
-cp ${OPENNMS_HOME}/etc/examples/opennms.properties.d/hosted_tsdb.timeseries.properties ${OPENNMS_HOME}/etc/opennms.properties.d/timeseries.properties
+org.opennms.timeseries.strategy=integration
```

Comment:
@mmahacek Freddy mentioned that other properties were set for `off-heap storage` feature (https://docs.opennms.com/meridian/2023/deployment/minion/off-heap-storage.html).